api/src/main/java/org/openmrs/module/ModuleClassLoader.java

```diff
@@ -530,8 +530,9 @@ protected void modulesSetChanged() {
 	 */
 	public void dispose() {
 		if (log.isDebugEnabled())
+		{
```

Comment:
For readability in Java, the brace "{" is on the same line as the previous line of code. If some people write on the same line, the other part in the new line, the code will become less readable.